### PR TITLE
Fix tests and other minor changes

### PR DESCRIPTION
**Pull request by roks0n**
**Changes:**
- Made tests work in PY2 and PY3 (the dongle test fails, because I don't have a dongle)
- updated `requirements.txt`
- other minor changes
- removed code from `TestUtilDef` because those functions still seem to be under development

### CODE REVIEW

File: arky/net/lisk.net [hunk 1]
```diff
@@ -12,10 +12,18 @@
     "port" : "8000"
   },
 
+  "seeds": [
+    "https://main.liskwallet.io",
+    "https://wallet.lisknode.io",
+    "https://wallet.mylisk.com",
+    "https://example124124asdfasdgadsg.com"
+  ],
+  "broadcast": 10,
```
Comment (Moustikitos) [inline arky/net/lisk.net]: On lisk network, broadcast is not to be used because a tx can't be sent to diff peers

**Comment by Moustikitos** [comment]
Good work !

- i still prefer to keep the comprehension list on peer selection (python comprehension list are very optimized and efficient)
- no broadcast parameter needed for lisk.net file

Please have a look to the review

File: arky/net/lisk.net (hunk 1 above)
Comment (roks0n) [reply arky/net/lisk.net]: What does this mean exactly? That we always need to send tx's to the same peer? Should I only add one `seed`?

**Comment by roks0n** [comment]
Tnx!

- It's only better because it doesn't call `append()`, but other than that it's the same. Performance improvement using list comprehension is really minimum. I'd argue that readability > perf. improvements in this case, just because they are so tiny.
- I've asked about the broadcast param on https://github.com/ArkEcosystem/arky/pull/66#discussion_r161319617

**Comment by Moustikitos** [comment]
I was so proud of my list comprehension :(

**Comment by Moustikitos** [comment]
Good work !

**Comment by boldninja** [comment]
@roks0n can you provide me with your ARK address or contact me on Slack (same username) so I can distribute your bounty?

**Pull request merged**